web/app/telegram_users_view.py

```diff
@@ -1,89 +1,109 @@
 from flask_restful import Resource, fields, marshal_with, abort,  reqparse
 from app import models
 from app import db
+from flask_jwt_extended import create_access_token, jwt_required, get_jwt_identity
+from flask import jsonify
 
 
 class RoleItem(fields.Raw):
```

Comment:
Maybe we need to replace RoleItem and StatusItem with some class like EnumItem or so.